src/System.Configuration.ConfigurationManager/ref/System.Configuration.cs

```diff
@@ -1435,6 +1435,9 @@ public partial class DelegatingConfigHost : System.Configuration.Internal.IInter
         public virtual void VerifyDefinitionAllowed(string configPath, System.Configuration.ConfigurationAllowDefinition allowDefinition, System.Configuration.ConfigurationAllowExeDefinition allowExeDefinition, System.Configuration.Internal.IConfigErrorInfo errorInfo) { }
         public virtual void WriteCompleted(string streamName, bool success, object writeContext) { }
         public virtual void WriteCompleted(string streamName, bool success, object writeContext, bool assertPermissions) { }
+#if netfx
```

Comment:
No, this is a leftover of when we weren't sure that we would actually have the relevant security types in standard. Feel free to add a no-op or PNS implementation for these.